tests/Functional/Command/ParallelCommandTest.php

```diff
@@ -81,9 +82,12 @@ public function testExecution(): void
         $this->assertContains(RaisingNoticeTestStub::class, $output);
         $this->assertContains(MissingProviderTestStub::class, $output);
         $this->assertContains(MySQLDeadLockTestStub::class, $output);
+        if (strpos((string) phpversion(), '7.1') !== 0) {
+            $this->assertContains(SessionTestStub::class, $output);
+        }
```

Comment:
Why is this needed?

Reply:
PHP 7.1 behaved differently in CLI mode with regard to session_id(), headers_sent() etc.

I'm investingating it further - maybe this will resolve to changes in test class setUp() (and no PHP version checks elsewhere).

Reply:
Alessandro, after investigation I have no other ideas to differentiate PHP versions behaviour than proposed in this PR. Please feel free to modify it;)